pkgs/applications/networking/qusb2snes/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, mkDerivation, git, qmake, qtbase, qtserialport, qtwebsockets }:
+mkDerivation rec {
```

Minor spacing thing.
```suggestion
{ lib, mkDerivation, git, qmake, qtbase, qtserialport, qtwebsockets }:

mkDerivation rec {
```